### PR TITLE
RSA keys are sometimes deprecated - actualize the doc

### PR DESCRIPTION
**Pull request by AronNovak**
**Note:** Please fill out the PR Template to ensure proper processing. (Edited to conform)

Closes: NA

## Summary

**[Generate and Add SSH Keys](https://pantheon.io/docs/ssh-keys)** - Fedora 33 broke the ability to push / pull from Pantheon: https://www.reddit.com/r/Fedora/comments/jhxbdh/no_ssh_public_key_auth_after_upgrade_to_fedora_33/

## Effect
(Use this section to detail the changes summarized above, or remove if not needed)
The following changes are already committed:
 - actualized the documentation with a workaround

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by AronNovak** [comment]
Of course supporting `ecdsa` keys would be even better, but that's not a documentation work.

**Comment by alexfornuto** [comment]
Additional context:
- https://fedoraproject.org/wiki/Changes/StrongCryptoSettings2
- https://dev.to/bowmanjd/upgrade-ssh-client-keys-and-remote-servers-after-fedora-33-s-new-crypto-policy-47ag

tl;dr: Fedora removed RSA algorithm as a valid key. You can update your key, or adjust your settings to allow RSA.

**Comment by alexfornuto** [comment]
Additionally, we should link to SSH-Keys in 'before you begin' for [Local Development](https://pantheon.io/docs/local-development)

File: source/content/ssh-keys.md [hunk 1]
```diff
@@ -126,3 +126,9 @@ To resolve, add your SSH key to the ssh-agent using the following command, repla
 ```bash{promptUser: user}
 ssh-add ~/.ssh/id_rsa
 ```
+
+Also if you use a recent Linux distribution (for example Fedora 33+), make sure RSA keys are enable in `~/.ssh/config`:
```
Comment (AronNovak) [inline source/content/ssh-keys.md:130]:
```suggestion
Also if you use a recent Linux distribution (for example Fedora 33+), make sure RSA keys are enabled in `~/.ssh/config`:
```

**Comment by alexfornuto** [comment]
Hi @AronNovak, I was wondering if you could run one more test for me: Would the current Fedora 33+ policies accept an RSA key with a bit size >= 2048? That seems to be indicated on https://fedoraproject.org/wiki/Changes/StrongCryptoSettings2 :

image

**Comment by alexfornuto** [comment]
Internal Reference: Coordinate with PROD-233

**Comment by AronNovak** [comment]
@alexfornuto This was my first attempt, even a 4096+ long key was not accepted.

**Comment by AronNovak** [comment]
That's probably a bug.
